packages/opencensus-core/test/test-console-exporter.ts

```diff
@@ -75,12 +63,11 @@ describe('ConsoleLogExporter', () => {
     it('should end a span', () => {
       const exporter = new ConsoleExporter(defaultBufferConfig);
       tracer.registerSpanEventListener(exporter);
-      // const rootSpan = new RootSpan(tracer);
-      const rootSpans = createRootSpans();
-      for (const rootSpan of rootSpans) {
+
+      tracer.startRootSpan({name: 'testRootSpans'}, rootSpan => {
```

Comment:
I know this wasn't part of the change, but I don't think we need to use a `Tracer` object to ensure that a `ConsoleLogExporter` works. You can just call `onEndSpan` directly rather than having it called through when the span is ended.

Reply:
Makes sense to me, updated to your suggestion.